imports/plugins/core/checkout/server/methods/workflow.js

```diff
@@ -237,7 +237,7 @@ Meteor.methods({
     // get index of `newWorkflowStatus`
     const resetToIndex = workflow.indexOf(newWorkflowStatus);
     // exit if no such step in workflow
-    if (!~resetToIndex) return false;
+    if (resetToIndex < 0) return false;
```

Comment:
@aldeed I've updated this file with your suggestions. I've returned the `workflow.includes` to `workflow.indexOf` and updated the `if` on the next line